src/utils.ts

```diff
@@ -68,24 +68,25 @@ export function cleanseSeparators(str: string): string {
 		.trim();
 }
 
-function getAnimeQuery(name: string, isVideo: boolean | undefined): string | null {
-	// Must be done after episode, season, and movie regex to be accurate
-	if (!isVideo) return null;
+export function getAnimeQueries(name: string): string[] {
+	// Only use if getTag returns anime as it's conditional on a few factors
+	let animeQueries: string[] = [];
 	const { title, altTitle, release } = name.match(ANIME_REGEX)?.groups ?? {};
```

Comment:
what is `release`?

Reply:
Usually episode but it can match other numbers so I didn't want to make promises.